src/System.CommandLine/CommandLineConfiguration.cs

```diff
@@ -125,22 +142,43 @@ private void AddGlobalOptionsToChildren(Command parentCommand)
             }
         }
 
+        /// <summary>
+        /// Represents all of the symbols to parse.
+        /// </summary>
         public ISymbolSet Symbols => _symbols;
 
+        /// <summary>
+        /// Represents all of the argument delimiters.
+        /// </summary>
         public IReadOnlyList<char> ArgumentDelimiters => ArgumentDelimitersInternal;
 
         internal IReadOnlyList<char> ArgumentDelimitersInternal { get; }
      
+        /// <summary>
+        /// Gets whether directives are enabled.
+        /// </summary>
         public bool EnableDirectives { get; }
 
+        /// <summary>
+        /// Gets whether POSIX bundling are enabled.
```

Comment:
```suggestion
        /// Gets whether POSIX bundling is enabled.
```

Reply:
Let's just say it was late at night :) 
Pushed the change.